src/scheduler/get_4byte.c

```diff
@@ -66,41 +60,33 @@
  * @brief
  * 		Gets the Scheduler Command sent by the Server
  *
- * @param[in]	sock	-	socket endpoint to the server
- * @param[in]	val	-	pointer to the value of the scheduler command sent.
- * @param[in]	jid	-	if 'val' obtained is SCH_SCHEDULE_AJOB, then '*jid'
- *						holds the jobid of the job to be scheduled.
+ * @param[in]	fd	-	connection to server
+ * @param[in/out]	cmd	-	sched command received from server
+ *
  * @return	int
- * @retval	0	: for EOF,
+ * @retval	0	: for EOF
  * @retval	+1	: for success
- * @retval	-1	: for error.
+ * @retval	-1	: for error
  *
- * @note
- *		Callers need to free the value pointed to by 'jid' after returning
- *		from this call.
  */
-
 int
-get_sched_cmd(int sock, int *val, char **jid)
+get_sched_cmd(int fd, sched_cmd_t *cmd)
```

Comment:
I think we better call it as sock only instead of fd as it is not a file descriptor.

Reply:
why? there is no difference between socket and file descriptor...

Reply:
As get_sched_cmd changed to take `svr_t *`, this comment becomes invalid now.